patches/chromium/revert_mojo_force_blink_headers_to_only_be_included_in_blink.patch

```diff
@@ -0,0 +1,63 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: David Sanders <dsanders11@ucsbalum.com>
+Date: Wed, 30 Oct 2024 23:28:24 -0700
+Subject: Revert "[mojo] Force blink headers to only be included in blink"
+
+This reverts commit 83b7fc2ecb7341531fdd346a905618f6fab06d12.
+
+Revert this change until we determine the best way to land it in Electron.
+
+Refs: https://chromium-review.googlesource.com/c/chromium/src/+/5962696
```

Comment:
Oh this is a interesting change, we should probably try with https://github.com/electron/electron/blob/15151c68533f5d8d1c9b57dbd7953e805f7719c9/shell/common/BUILD.gn#L20 as I don't see us using any `mojom-blink.h` headers. Created https://github.com/electron/electron/issues/44499

Reply:
@dsanders11 can you try this in this PR?

Reply:
@jkleinsc, I'm personally more inclined to leave it for a follow-up just so we can get this one merged, since it's ready and a new person is picking up the roll next week.